policyengine_us/variables/gov/states/nj/tax/income/credits/property_tax_credit/nj_property_tax_credit_eligible.py

```diff
@@ -37,14 +37,8 @@ def formula(tax_unit, period, parameters):
             | senior_head_or_spouse
         )
 
-        federal_agi = tax_unit("adjusted_gross_income", period)
-        filing_status = tax_unit("filing_status", period)
-        agi_eligible = federal_agi <= p.income_limit[filing_status]
```

Comment:
Does this mean that we need to remove the parameter file or do we use it elsewhere?

Reply:
We use it for `nj_property_tax_deduction_eligible`

Reply:
Ok makes sense, so these are the only changes we need?

Reply:
Yes